core/assets/js/tabbar.js

```diff
@@ -0,0 +1,109 @@
+let activeTabId = ""
+
+export const Tabbar = {
+    mounted() {
+        var initialTab = "tab_"+this.el.getAttribute("initial-tab")
+        this.show(initialTab, true)
+
+    },
+    updated() {
+        this.show(activeTabId, false)
+    },
+
+    show(nextTabId, scrollToTop) {
+        activeTabId = nextTabId
+        var tabs = Array.from(document.querySelectorAll('[id^="tab_"]'));
+
+        if (tabs.filter((tab) => { return tab.id === nextTabId }).length === 0) {
+            // skip unknown tab
+            return
+        }
+
+        // Show active tab
+        tabs.forEach(tab => {
+            var tab_id = tab.getAttribute("id")
```

Comment:
Why `getAttribute` here? The code above used `.id`.

Reply:
No specific reason

Reply:
Changed in my next PR